### PR TITLE
feat/add nextjs react query sample

### PR DESCRIPTION
**Pull request by brunos3d**
This PR aims to add an advanced sample of usage of react-query with Next.js and Module Federation.

### CODE REVIEW

File: nextjs-ssr-react-query/apps/button-e2e/cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'cypress';
```
Comment (PeaceDate) [inline nextjs-ssr-react-query/apps/button-e2e/cypress.config.ts]: Do you really need this e2e folders? 
`button-e2e`
`header-e2e`
`host-e2e`
`pip-e2e`

File: package.json [hunk 2]
```diff
@@ -64,6 +64,7 @@
       "./nextjs-react/*",
       "./nextjs-ssr",
       "./nextjs-ssr/*",
+      "./nextjs-ssr-react-query",
```
Comment (PeaceDate) [inline package.json:82]: You need to add here name of your root package.json. 
So, for now you have two ways, update your sample root package.json
`/module-federation-examples/nextjs-ssr-react-query/package.json`
to

```
{
  "name": "nextjs-ssr-react-query",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {},
  "private": true,
  "dependencies": {
    "@module-federation/nextjs-mf": "^6.1.3",
    //...
```

or  add here name from package.json witch you already have, it's `acme`

But I prefer first method, it's more clear😀

**Review by PeaceDate — CHANGES_REQUESTED** [review]

File: nextjs-ssr-react-query/apps/button-e2e/cypress.config.ts (hunk 1 above)
Comment (brunos3d) [reply nextjs-ssr-react-query/apps/button-e2e/cypress.config.ts]: No, they are not important, I removed it here, thanks :handshake: :rocket:

File: package.json (hunk 2 above)
Comment (brunos3d) [reply package.json:82]: Solved :rocket:

**Review by PeaceDate — APPROVED** [review]

**Review by CaioRolla — APPROVED** [review]

**Pull request merged**